### PR TITLE
#500 Add annotation to podSpec

### LINKED ISSUE #500: Add Annotations to keycloak pod
### Description

Add Annotations to keycloak pod

### Discussion

_No response_

### Motivation

Actually we can only use labels for keycloak pod, unfortunately when you want to use multus annotations are necessary

### Details

_No response_

### PR DESCRIPTION
**Pull request by bovigny**
Signed-off-by: bovigny <bovignyc@gmail.com>

## Related GH Issue
Closes #500 

## Additional Information
Add annotations to podSpec

## Checklist:
- [x] Automated Tests
- [x] [Documentation](https://github.com/keycloak/keycloak-documentation) changes if necessary
- [x] (If required) Discussed on [Keycloak DEV](https://groups.google.com/forum/#!forum/keycloak-dev)

### CODE REVIEW

**Comment by Stefanqn** [comment]
Linting fails.

File: pkg/model/constants.go [hunk 1]
```diff
@@ -64,3 +64,4 @@ const (
 )
 
 var PodLabels = map[string]string{}
+var PodAnnotations = map[string]string{}
```
Comment (vmuzikar) [inline pkg/model/constants.go:67]: We don't need this (unlike for Labels).

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@bovigny Thank you for your contribution! Added a few comments.

**Comment by vmuzikar** [comment]
@bovigny Hi, just wanted to ask if this is still something you want to work on. We now have a very similar PR #531. Maybe we can close this one in favor of #531.

**Comment by vmuzikar** [comment]
I'm closing this as #531 was merged.

**Pull request closed without merging**